PR-Trigger/index.js

```diff
@@ -36,7 +37,8 @@ const httpTrigger = async function (context, _req) {
     const acceptedEventsToActions = {
         "pull_request": ["opened", "closed", "reopened", "edited", "synchronized", "ready_for_review"],
         "pull_request_review": ["submitted", "edited", "dismissed"],
-        "issue_comment": ["created", "edited", "deleted"]
+        "issue_comment": ["created", "edited", "deleted"],
+        "status": ["*"]
     }

```

Comment:
We allowlist the events and actions from GH webhooks, so that it's not doing too much work.